tests/coafileTest.py

```diff
@@ -0,0 +1,27 @@
+import unittest
+import re
+from coalib.misc.Shell import run_shell_command
+from coalib.output.printers.LogPrinter import LogPrinter
+from pyprint.ConsolePrinter import ConsolePrinter
+from coalib.settings.ConfigurationGathering import load_config_file
+from coalib.output.printers.LOG_LEVEL import LOG_LEVEL
+
+
+class coafileTest(unittest.TestCase):
+
+    def setUp(self):
+        self.logprinter = LogPrinter(ConsolePrinter(), LOG_LEVEL.ERROR)
```

Comment:
@Mixih So it does not work. logging.getLogger does not have warn and err methods,it has warning and error methods instead.
I think a better solution is to check the file exists beforehand,making the need for a logger unnecessary in load_config_file.